base_agent/sql_memory.py

```diff
@@ -29,7 +29,17 @@
     NODELIST,
 )
 
-
+NONPICKLE_ATTRS = [
```

Comment:
why are these nonpickle-able?

Reply:
there are a lot of pieces of Tasks and other memories that are not pickleable for various reasons (e.g. connection to the database); most of these predate this PR or change.  I just grouped them into an explicit const rather than a separate list in each function, like in https://github.com/facebookresearch/droidlet/blob/main/base_agent/sql_memory.py#L1139.